src/integrationTest/java/uk/gov/hmcts/reform/iataskconfiguration/CamundaTaskInitiationTest.java

```diff
@@ -267,12 +225,17 @@ void given_single_rule_match_should_evaluate(String eventId,
         if (expectedTaskCategoryPresent) {
             assertThat(singleResult.getEntry("taskCategory"), is(taskCategory));
         }
-        assertThat(singleResult.containsKey("delayDuration"), is(false));
+        if(expectedDelayDuration) {
+            assertThat(singleResult.getEntry("delayDuration"), is(delayDuration));
+        } else {
+            assertThat(singleResult.containsKey("delayDuration"), is(false));
+        }
         if (workingDaysAllowed > 0) {
             assertThat(singleResult.getEntry("workingDaysAllowed"), is(workingDaysAllowed));
         } else {
             assertThat(singleResult.containsKey("workingDaysAllowed"), is(false));
         }
+
```

Comment:
Given the AC's, can we include an assertion on the role/group? i.e, check the TCW.